src/metabase/api/permissions.clj

```diff
@@ -82,8 +81,9 @@
         (throw (ex-info (tru "Cannot parse permissions graph because it is invalid: {0}"
                              (pr-str explained))
                         {:status-code 400
-                         :error explained
-                         :humanized (me/humanize explained)}))))
+                         :error (str (me/humanize explained)
```

Comment:
This will put the shorter (readable) error front and center if there is an error with the shape of this data.